runtime/onert/core/src/ir/train/UseDefGenerator.cc

```diff
@@ -71,6 +71,29 @@ UseDefChains UseDefGenerator::operator()()
   return _training_usedefs;
 }
 
+void UseDefGenerator::visit(const train::operation::ElementwiseActivation &node)
+{
+  if (node.param().op_type != operation::ElementwiseActivation::Type::RELU)
+  {
+    throw std::runtime_error{"UseDefGenerator: Not yet supported activation type"};
+  }
+  assert(_node_to_idx.find(&node) != _node_to_idx.end());
+  const auto &op_index = _node_to_idx.at(&node);
+  const auto backwarding_op_index = TrainingOperationIndex{op_index, false};
+
+  // Insert use of forwarding output
+  const auto &out_index = node.getOutputs().at(0);
+  const auto out_forwarding_index = TrainingOperandIndex{out_index, true};
+  insertUse(out_forwarding_index, backwarding_op_index);
+
+  // Set def of backwarding(backprop) inputs
```

Comment:
Maybe..? 

```suggestion
  // Set def of backwarding(backprop) output
```

Reply:
Ah, This comment is intentional. 
I checked the other PR's comments.

This might be my confusion from the word's meaning.